src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -167,54 +168,40 @@ export default {
                 this.changeFeatureTitle({ feature: this.feature, title: value })
             },
         },
-        geometry: {
-            get() {
-                /*
+        geometry() {
+            /*
                 Openlayers polygons coordinates are in a triple array
                 The first array is the "ring", the second is to hold the coordinates, which are in an array themselves
                 We don't have rings in this case, so we need to create an ol geometry
                 */
 
-                const geom = [this.feature.coordinates]
+            const geom = [this.feature.coordinates]
 
-                return new Polygon(geom)
-            },
+            return new Polygon(geom)
         },
-        isClosed: {
-            get() {
-                return (
-                    this.feature.coordinates[0][0] ===
-                        this.feature.coordinates[this.feature.coordinates.length - 1][0] &&
-                    this.feature.coordinates[0][1] ===
-                        this.feature.coordinates[this.feature.coordinates.length - 1][1]
-                )
-            },
+        isClosed() {
+            return (
+                this.feature.coordinates.length > 3 &&
```

Comment:
a triangle has "4 points" because the last one is accounted for (We only remove the extra point in geodesic coordinates, not standard coordinates) . If I make this with greater than 2, it will give me the area of lines (which is 0, not a big deal, but still not an information we need)

I can put a small comment to make it clear in the code if you'd like